python-lib/dku_kube/gpu_driver.py

```diff
@@ -29,8 +29,9 @@ def add_gpu_driver_if_needed(cluster_id, kube_config_path, connection_info, tain
     # Retrieve the tolerations on the daemonset currently deployed to the cluster.
     if has_gpu_driver(kube_config_path):
```

Comment:
should add the same for autoscaler

Reply:
Not necessary as the autoscaler is a pod that needs to be deployed on one node. As long as the taints are added when the autoscaler is added the first time, it will be able to be deployed and work.